assets/src/domain/eventEditor/ui/ticketAssignmentsManager/components/ModalContainer.tsx

```diff
@@ -10,9 +10,9 @@ const ModalContainer: React.FC<ModalContainerProps> = ({ isOpen, onClose, ...pro
 
 	let title = '';
 	if (assignmentType === 'forDate') {
-		title = sprintf(__('Ticket Assignment Manager for Datetime: %s - %s'), entity.dbId, entity.name);
+		title = sprintf(__('Ticket Assignment Manager for Datetime: %s - %s'), String(entity.dbId), entity.name);
```

Comment:
Always better to use language constructs (which are faster) instead of calling functions. Although, it won't make a big difference here but still it's better.
```suggestion
		title = sprintf(__('Ticket Assignment Manager for Datetime: %s - %s'), `${entity.dbId}`, entity.name);
```

Reply:
I would prefer clear and clean code over minor performance optimization.
It's not the case here, but in order to know exactly what is the performance gain or loss, I'd have to measure both cases and see exact number. And, if I'd do that probably there wouldn't be very noticeable difference.

Reply:
However, let's take aside performance discussion.
I think there is a bit of the problem with `sprintf`.
As I understand `sprintf` is returning a string, then why I should care about these type coercions?
I think `sprintf` should be smart enough to this and maybe other check and I should just pass whatever I want. If I pass `undefined` or `null`, ignore it, otherwise do your smart job and please don't bother me with type coercion.

Reply:
I love the DX `classnames` have, I can pass whatever I want, it can be `null`, object or string and then it's smart enough to figure out by itself what needs to be the result.

Reply:
At some point, maybe, we should create our own wrapper for `sprintf` and make it a bit smarter.

Reply:
`classnames` is awesome:

```
classNames('foo', 'bar'); // => 'foo bar'
classNames('foo', { bar: true }); // => 'foo bar'
classNames({ 'foo-bar': true }); // => 'foo-bar'
classNames({ 'foo-bar': false }); // => ''
classNames({ foo: true }, { bar: true }); // => 'foo bar'
classNames({ foo: true, bar: true }); // => 'foo bar'

// lots of arguments of various types
classNames('foo', { bar: true, duck: false }, 'baz', { quux: true }); // => 'foo bar baz quux'

// other falsy values are just ignored
classNames(null, false, 'bar', undefined, 0, 1, { baz: null }, ''); // => 'bar 1'
```